site/_includes/partials/deprecations-table.njk

```diff
@@ -0,0 +1,36 @@
+{% from 'macros/icon.njk' import icon with context %}
+<filtered-table>
+  <div class="deprecations-calendar__rows">
+    <div class="deprecations-calendar__row filtered__empty" hidden>
+        Sorry, we couldn't find any results that match your selected filters. 
+    </div>
+    <div class="deprecations-calendar__row deprecations-calendar__header filtered__header">
```

Comment:
Similar as above, those are weird in BEM. Think about splitting them up into more meaningful, smaller blocks. For example `deprecations-calendar__row` can probably becaome `.filtered-item` then with `.filtered-item__header`, ...